src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -108,19 +121,63 @@ export class QuickBar extends LitElement {
       filter
     );
 
+    this._topEntityIdResult = entities[0];
+
     return html`
-      ${entities.map((entity_id) => {
-        const domain = computeDomain(entity_id);
-        return html`
-          <mwc-list-item @click=${this._entityMoreInfo} graphic="icon">
-            <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
-            ${entity_id}
-          </mwc-list-item>
-        `;
-      })}
+      <mwc-list activatable @selected=${this._entityMoreInfo}>
+        ${entities.map((entityId, index) => {
+          const domain = computeDomain(entityId);
+          return html`
+            <mwc-list-item
+              graphic="icon"
+              .entityId=${entityId}
+              .activated=${index === 0}
+              @keydown=${this._focusInputField}
+            >
+              <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
+              ${entityId}
+            </mwc-list-item>
+          `;
+        })}
+      </mwc-list>
     `;
   });
 
+  private _focusFirstElement(ev: KeyboardEvent) {
```

Comment:
```suggestion
  private _handleInputKeyDown_(ev: KeyboardEvent) {
```

Reply:
Assume you didn't intend the second underscore ;)

Done